---
bool useful
string color1
string color2
